velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -419,6 +417,13 @@ ExpressionFuzzer::ExpressionFuzzer(
   registerFuncOverride(
       &ExpressionFuzzer::generateRegexpReplaceArgs, "regexp_replace");
   registerFuncOverride(&ExpressionFuzzer::generateSwitchArgs, "switch");
+
+  // Init stats and register listener.
+  for (auto& name : supportedFunctions) {
+    exprNameToStats_.insert({name, ExprUsageStats()});
+  }
+  statListener_ = std::make_shared<ExprStatsListener>(exprNameToStats_);
+  exec::registerExprSetListener(statListener_);
```

Comment:
I have added a LOG at WARNING level in case this happens since ideally it should only be registered once. This only returns false if the listener is already registered which should be harmless in this case so a warning should be sufficient.